lib/qs/client.rb

```diff
@@ -34,6 +34,14 @@ def enqueue(queue, job_name, params = nil)
         job
       end
 
+      def push(queue_redis_key, serialized_payload)
```

Comment:
@jcredding I think I'd prefer `append` here.  It is the corresponding action to `prepend`.